lib/src/templates/title_slide.dart

```diff
@@ -57,10 +57,14 @@ class FlutterDeckTitleSlide extends StatelessWidget {
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            AutoSizeText(title, style: theme.titleTextStyle),
+            Expanded(
```

Comment:
`Expanded` must completely ruin the layout. If tests are failing because of the overflow error, replace these with `Flexible`.